src/main/java/com/tvm/DAO/TicketAndUserDAO.java

```diff
@@ -0,0 +1,39 @@
+package com.tvm.DAO;
+
+import com.tvm.Model.Ticket;
+import com.tvm.Model.User;
+import com.tvm.constants.Queries;
+import com.tvm.utils.DBUtil;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public class TicketAndUserDAO {
+
+    public void updateUserAndTicket(User user, Ticket ticket, int userId, int ticketId) throws SQLException {
+        String userQuery = Queries.UPDATE_USER;
+        String ticketQuery = Queries.UPDATE_TICKET;
```

Comment:
you can directly set Queries.UPDATE_USER to the method as a parameter during the call as this is a constant